### DISCUSSION TITLE
Could the Collection Health Index (CHI) be a Latimer Core term?

### DISCUSSION THREAD

**Discussion by ManonGros** [tdwg/ltc#99] [Integrations]
Hi Latimer Core people!

This is another question a bit similar to (https://github.com/tdwg/ltc/discussions/80). I was recently made aware of the Collection Health Index (CHI) which is used in some parts of the world (at least Colombia https://raccefyn.co/index.php/raccefyn/article/view/1159 - and New Zealand - https://riojournal.com/article/93841/) to assess the health of a collection.
It is based on the work of
> McGinley, R. J. (1993). Where´s the management in collection management? Planning for improved care, greater use, and growth of collections. Int. Sym. & First World Congress on Pres. and Cons. of Nat. Hist. Col., 3, 309-333. https://repository.si.edu/handle/10088/9736

and it is defined by 10 levels:

1. **Conservation problem**: Specimens deteriorating, potentially cullable, or unprepared. Collection unit in need of immediate attention: museum pests, rusting pins, crystal- lizing slide media, unringed Hoyer7smedia, evaporated alcohol, fading labels, broken cover slip or slide, etc. Primary types mixed in general collection.
2. **Specimens unidentifiable, inaccessible**: Material properly prepared but not sorted or only rough-sorted;not readily avaiIable to specialists.
3. **Specimens unidentifiable, accessible**: Specimenssorted to a level necessaryto be efficientlyaccessibleto research specialists for study. All specimens in soft-bottom trays, shell vials in jars, or slide boxes, with appropriate labels denoting pertinent taxonomic information. Taxonomic category may vary among different taxa.
4. **Specimens identified but not integrated into collection**: Valuable material that has been identified to t h e species level but not yet put away - effectively inaccessible.
5. **Specimens identified but curation incomplete**:  specimens identified and integrated. However, this collection unit needs upgrading, e.g., names checked, header labels for unit trays prepared, transfer to soft-bottom trays, etc., box header labeIs checked against slides. Alcoholic collection with individual exposed viaIs, jar labels misleading or without detail, overall format heterogenous.
6. **Specimens identified and properly curated in accordance with departmental collection standards**: All specimens in soft-bottom unit trays; all unit-tray header labels, drawer at box, and cabinet labels completed; space left for expansion. When a collection reaches LEVEL6, the names(s) of who did the curation,the date of curation and listing of catalogues or monographs on which the names are based should be entered in the Smithsonian Collection Inventory File (CIF) computerized files. Vial within jar system established, labels complete and accurately typed, expansion space allocated, and alcohol levels adequate.
7. **Data capture: species level inventory**: Species inventory with header or box label generation completed. Alcoholic collection with taxa listed on jar labels and in database. As in LEVEL 6, plus species-level inventory.
8. **Data capture: specimen label data capture**: Label data are recorded for systematic, biogeography or natural history studies. The data elements recorded will vary by group, however, a standard core set includes the following: a)country/province/place/latitude- longitude; b) date of collection; c) collector(s); d) miscellaneous (for example, elevation, host). Miscellaneous voucher specimens, including types, from various studies are also ranked at this level.
9. **Data capture: research data capture**: Measurements, graphics, or other pertinent (descriptions) of specimens are captured in a database.
10. **Scientific voucher material**: Groups of specimens included in published monographic, synthetic, or revisionary works, including primary type collection if storage is at LEVEL 7 or higher. When groups of specimens serve to voucher published scientific collection-based studies and have undergone conservation and accession requirements of at least LEVEL 7.

I was told that it would be useful to have CHI as collection descriptors in GRSciColl so that collection entries can be found based on their CHI level.
Ideally, there would be a field dedicated for CHI. Is that something that could make sense?

**Comment by debpaul** [comment]
Cool Idea @ManonGros. I think perhaps we need a conversation to consider there are different **health index** systems and make a general field/s that support sharing these data.

This standard (C Favret, et al 2007) builds on the work of McGinley (also an author on the Favret paper) and keeps the issues noted in the assessment -- separate (rather than adding them together as McGinley does). 

- Favret C, Cummings KS, McGinley RJ, Heske EJ, Johnson KP, Phillips CA, Phillippe LR, Retzer ME, Taylor CA, Wetzel MJ. **Profiling natural history collections: a method for quantitative and comparative health assessment**. In Collection Forum 2007 (Vol. 22, No. 1-2, pp. 53-65). [Favret_et-al_2007.pdf](https://github.com/user-attachments/files/17478620/Favret_et-al_2007.pdf)

The Entomological Collections Network (worldwide reach) teaches the use of this standard in its collection management workshops.

**Considerations**
- The McGinley system (designed originally for entomological collections) is _usually_ used to assess a **drawer** of insects or some other "storage unit" level. So the score given is for a drawer or jar or box of items (not a specimen). 
- The Favret et al authors created a more robust profiling system (see the paper for the history and details).
- We'd need some way to provide
  - what **scale_used** and **score** 
- I'm unsure of how to report this for an entire collection (for grscicoll). The Favret paper shows examples of the data that can be generated  (scores for a given collection, by different **assessment variables** (e. g. conservation, data quality, container, labels, arrangement, identification, computerization, processing).
  - So, I think, kind of collection, assessment variable, scores for that variable, scope of that variable, assessment scale / method used.

@tcmcelrath @mjyoder

**Comment by ManonGros** [comment]
Thanks @debpaul!
I wasn't aware of the CHI until recently (and I am not familiar with health index in general) so the input is much appreciated!

For the question around the level (drawer, unit, collection) of the health index: my thinking would be that this would be shared as [collection descriptors](https://data-blog.gbif.org/post/grscicoll-collection-descriptors/) (so smaller scale than a collection). For example, for a set of specimens belonging to a taxonomic group or a unit. This would fit in the Latimer Core scope and would work for GRSciColl as well.

Latimer Core recommends using measurmentsOrFacts for these types of metrics. The problem is that I can't keep track of all the metrics that can be used for describing collections and I can't easily make collections searchable if they all have slightly different metrics. That's why I have been proposing that more globally used metrics (like the CHI or MIDS level) be defined in the standards (in this care Latimer Core).
So if there was a more generic set of terms for health indexes that would encompass the **scale_used** as well as the **score**, it would be really helpful to have the score and scales defined in a standard.

**Comment by ColinFavret** [comment]
Hi @ManonGros , I'm not familiar with the ltc, but I can see that this issue of CHI would almost certainly require more than one new field. That said, I don't have anything to add to what @debpaul already wrote. Sounds like a nice initative!

**Comment by fmjjones** [comment]
Hi @ManonGros, We are implementing the standard New term template and Change Request template that DarwinCore uses to track suggestions for review. Please fill out the New Term template https://github.com/tdwg/ltc/blob/main/.github/ISSUE_TEMPLATE/new-term-template.md as completely as possible. This helps us begin the conversation of next steps with the suggestion. Plus, you'll get the honor of being the first person to use the template ;)
